src/styles/base.css

```
@@ -31,9 +31,9 @@
 		--sp-s: 8px;
 	}
 
-	--english-font: "Noto Sans", "Robot", "Avenir Next", "Avenir",
-		"Century Gothic", "SF Pro Text", "Arial", -apple-system, BlinkMacSystemFont,
-		sans-serif;
+	--english-font:
+		"Noto Sans", "Robot", "Avenir Next", "Avenir", "Century Gothic",
+		"SF Pro Text", "Arial", -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.3.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)